docs/Deploy/single-sign-on-sso.md

```diff
@@ -109,7 +109,7 @@ After supplying the above information, your IdP will generate an issuer URI and
 
 ## Fleet configuration
 
-To configure SSO in Fleet head to **Settings > Organization settings > Single sign-on options**.
+To configure SSO in Fleet head to **Settings > Integrations > Single sign-on (SSO)**.
```

Comment:
This change already exists on the `docs-v4.67.0` branch, I have no idea why it's showing up in this diff. Please ignore.